code/modules/mob/living/carbon/carbon.dm

```diff
@@ -768,6 +768,8 @@
 
 		for(var/datum/disease/D in viruses)
 			D.cure(0)
+			
+	regenerate_limbs()

```

Comment:
Alright well I still want wands/staves of healing to fix this so I guess I'll just throw that bit in the bullet